packages/db/test/unit/controller/level.test.ts

```diff
@@ -144,7 +144,8 @@ describe("LevelDB controller", () => {
           return "gdu";
         }
       } catch {
-        /* no-op */
+        /* eslint-disable no-console */
+        console.error("Cannot find correct gdu command");
```

Comment:
How do you feel about saying

```suggestion
        console.error("Cannot find gdu command, falling back to du");
```